pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -1203,10 +1188,26 @@ func (lbaas *LbaasV2) EnsureLoadBalancerDeleted(clusterName string, service *v1.
 	return nil
 }
 
+func (lbaas *LbaasV2) IsNotFound(err error) bool {
```

Comment:
Yes, caller don't know getxxxx() return ErrNotFound or isNotFound err, we can use IsNotFound() to check err.